### PR TITLE
LEAN-3958 EI25Q1-010 - Implement texts for Track changes

### PR DESCRIPTION
**Pull request by fwaisi**

### CODE REVIEW

File: src/schema/nodes/footnotes_element.ts [hunk 1]
```diff
@@ -28,7 +26,13 @@ export interface FootnotesElementNode extends ManuscriptNode {
   attrs: Attrs
 }
 
-export const footnotesElement: NodeSpec = {
+interface FootnotesElementNodeSpec extends ManuscriptNodeSpec {
+  isWrapper: true
```
[the remark this answers is not in the document]
Comment (fwaisi) [reply src/schema/nodes/footnotes_element.ts]: I need it to distinguish between normal nodes and wrapper nodes to handle the displayed content. This approach is much easier—without it, I'd have to add more switch cases for each wrapper node.

[the remark this answers is not in the document]
Comment (IslamJMomani) [reply src/schema/nodes/footnotes_element.ts]: However, most of the nodes are actually wrappers, and you only defined `footnotesElement` and `tableElementFooter` as wrappers.

If you need special handling for these nodes, you can implement that on the frontend - without adding more switch cases.

[the remark this answers is not in the document]
Comment (IslamJMomani) [reply src/schema/nodes/footnotes_element.ts]: By the way, you can use `isElementNodeType` to distinguish wrapper nodes, but adding `isWrapper` feels redundant to me.

[the remark this answers is not in the document]
Comment (fwaisi) [reply src/schema/nodes/footnotes_element.ts]: Yes, its very helpful. Thanks.

**Review by IslamJMomani — APPROVED** [review]

**Comment by IslamJMomani** [comment]
@fwaisi, I'm okay with this and have just approved it. I was thinking—though I don't have a strong opinion—why not create a mapping between the nodes and their names? Check `nodeNames`.

@hhusban, did you see any advantages of one option over the other? Personally, I tend to prefer the mapping approach.

**Comment by hhusban** [comment]
I do agree with Islam, the mapping would be centralize, the only issue with this approach its a bit forgettable since there is no enforcing/validating on it

**Comment by fwaisi** [comment]
> @fwaisi, I'm okay with this and have just approved it. I was thinking—though I don't have a strong opinion—why not create a mapping between the nodes and their names? Check `nodeNames`.
> 
> @hhusban, did you see any advantages of one option over the other? Personally, I tend to prefer the mapping approach.

@IslamJMomani done.

**Review by IslamJMomani — APPROVED** [review]

**Pull request merged**